public/pages/ReviewAndCreate/components/DetectorDefinitionFields/DetectorDefinitionFields.tsx

```diff
@@ -94,19 +94,36 @@ export const DetectorDefinitionFields = (
         !props.validDetectorSettings &&
         props.validationResponse.hasOwnProperty('message')
       ) {
-        return (
-          <EuiCallOut
-            title="Issues found in the detector settings"
-            color="danger"
-            iconType="alert"
-            size="s"
-            style={{ marginBottom: '10px' }}
-          >
-            <ul>
-              <li>{props.validationResponse.message}</li>
-            </ul>
-          </EuiCallOut>
-        );
+        if (isEqual(props.validationResponse.validationType, 'model')) {
+          return (
+            <EuiCallOut
+              title="We identified some areas that might improve your model"
+              color="warning"
+              iconType="iInCircle"
+              size="s"
+              style={{ marginBottom: '10px' }}
+            >
+              {JSON.stringify(props.validationResponse.message).replace(
```

Comment:
For consistency, should this be in a bullet-point format like it is for blocker issues on lines 121-123?

Reply:
Or will only one possible message be returned per non-blocker failure?

Reply:
Only one possible message will be returned per non-blocker failure as of now so thought it looks nicer without the bullet point